contracts/gateway/EIP20Gateway.sol

```diff
@@ -572,18 +572,18 @@ contract EIP20Gateway is GatewayBase {
     {
         require(
             _messageHash != bytes32(0),
-            "Message hash must not be zero"
+            "Message hash must not be zero."
         );
         require(
             _rlpParentNodes.length > 0,
-            "RLP parent nodes must not be zero"
+            "RLP parent nodes must not be zero."
         );
 
         // Get the message object
         MessageBus.Message storage message = messages[_messageHash];
         require(
             message.intentHash != bytes32(0),
-            "StakeIntentHash must not be zero"
+            "StakeIntentHash must not be zero."
```

Comment:
Please add a period to L586.

Reply:
Is it not already added ?

Reply:
![image](https://user-images.githubusercontent.com/1196244/51469962-dcd5ff00-1d72-11e9-82f7-cfd449b9e988.png)
I can see it 🤔